mgr_test.go

```diff
@@ -74,6 +79,39 @@ func TestManagerAddNode(t *testing.T) {
 		if err != nil && err.Error() != test.err {
 			t.Errorf("mgr.AddNode(%s, %d) = %s, expected %s", test.addr, test.id, err.Error(), test.err)
 		}
+	}
+}
 
+// Reusing tests/unresponsive/unresponsive.proto definitions
```

Comment:
Maybe we should have a dedicated package for this kind of testing?

Reply:
Added new `tests/dummy/dummy.proto` to be used for such testing. Please let me know if you have suggestions for a better name.